docs/extending/developing-code/extending/documentation-markup.rst

```diff
@@ -126,10 +132,6 @@ It is good to create examples that convey some aspect about the Mathics3 Functio
 In the past, the documentation system was abused and ran edge cases
 and prior bugs fixed. For that please write a pytest.
 
-
-We have not purged yourself of this behavior, so will find following
```

Comment:
I am not sure about understanding this phrase. I rephrase this below, where the special markup syntax for doctests is presented. Notice that almost all the doctests which were not suitable for documentation were already moved out to the pytest system.

Reply:
The rephrase is right and covers what I think I meant.